classes/manage_content_types.php

```diff
@@ -88,6 +89,31 @@ protected function set_external_page(): void {
         admin_externalpage_setup('mod_cms/managetypes');
     }
 
+    /**
+     * Adds breadcrumbs depending on the action being taken.
+     *
+     * @param string $action
+     */
+    protected function add_breadcrumb(string $action) {
+        global $PAGE, $FULLME;
+
+        switch($action) {
+            case 'add':
+                $PAGE->navbar->add($this->get_new_heading());
+                break;
+            case 'edit':
+                // Need to add extra breadcrumbs when URL does not exactly match admin setting page URL.
+                foreach (['modcmsfolder', 'mod_cms/managetypes'] as $label) {
+                    if ($node = $PAGE->settingsnav->find($label, \navigation_node::TYPE_SETTING)) {
```

Comment:
`use navigation_node;`
```suggestion
                    if ($node = $PAGE->settingsnav->find($label, navigation_node::TYPE_SETTING)) {
```